src/components/layout/ChatArea.tsx

```diff
@@ -265,6 +292,39 @@ export const ChatArea: React.FC = () => {
     if (e.key === "Enter" && !e.shiftKey) {
       e.preventDefault();
       handleSendMessage();
+      ircClient.sendRaw(
+        selectedServerId ?? "",
+        `@+typing=done TAGMSG ${selectedChannel?.name ?? ""}`,
+      );
+    }
+  };
+
+  const handleUpdatedText = (text: string) => {
```

Comment:
I think we need to debounce here and limit to like once every 2 seconds or something instead of every key press.